src/main/java/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,17 @@
+package strategy;
+
+import db.Storage;
+import model.FruitTransaction;
+
+public class ReturnOperation implements Operating {
+    public ReturnOperation() {
```

Comment:
Here and in other operations, it looks like you do not need explicit empty constructor